### PR TITLE
Add variant: Generic H503KBU

### PR DESCRIPTION
**Pull request by dojyorin**
I'm sorry for inconvenience caused with PR:Nucleo-H503RB.
Could you please make sure added and modified code?

I set clock configuration in STM32CubeMX as described on Wiki, but peripheral configuration were not present in code.
(e.g. `PeriphClkInitStruct.UsbClockSelection = RCC_USBCLKSOURCE_HSI48`)
Looking at CubeMX clock configuration GUI, for example, USB uses HSI48 by default, so does that mean no additional configurations are required?

**Summary**

This PR implements the following **features**

* [x] New variant: Generic STM32H503KBU

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -540,6 +540,7 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 
 | Status | Device(s) | Name | Release | Notes |
 | :----: | :-------: | ---- | :-----: | :---- |
+| :yellow_heart:  | STM32H503KB | Generic Board | *2.9.0* |  |
```
Comment (fpistm) [inline README.md:543]:
```suggestion
| :yellow_heart:  | STM32H503KB | Generic Board | **2.8.1** |  |
```

**Review by fpistm — CHANGES_REQUESTED** [review]
Minor typo to fix and it is ok for me.
About Periphjeral clock config, no need to specify it as HSI48 is the default entry.

**Comment by dojyorin** [comment]
> About Periphjeral clock config, no need to specify it as HSI48 is the default entry.

Thank you for teaching me.
Version fix suggestion was accepted.

**Review by fpistm — APPROVED** [review]
LGTM

**Pull request merged**